app/docs/0.11.x/clustering.md

```diff
@@ -4,135 +4,54 @@ title: Clustering Reference
-
-<center>
-  <img src="/assets/images/docs/clustering.png" style="height: 400px"/>
-</center>
-
-Kong clustering settings are specified in the configuration file at the following entries:
-
-* [cluster_listen][cluster_listen]
-* [cluster_listen_rpc][cluster_listen_rpc]
-* [cluster_advertise][cluster_advertise]
-* [cluster_encrypt_key][cluster_encrypt_key]
-* [cluster_ttl_on_failure][cluster_ttl_on_failure]
-* [cluster_profile][cluster_profile]
-
-## 2. Node Discovery
-
-On startup, Kong will try to auto-detect the first private, non-loopback, IPv4 address and register the address into a `nodes` table in the datastore to be advertised to any other node that's being started with the same datastore. When another Kong node starts it will then read the `nodes` table and try to join at least one of the advertised addresses.
-
-Once a Kong nodes joins one other node, it will automatically discover all the other nodes thanks to the underlying gossip protocol.
-
-Sometimes the IPv4 address that's automatically advertised by Kong it's not the correct one. You can override the advertised IP address and port by specifying the [cluster_advertise][cluster_advertise].
-
-## 3. Network Requirements
-
-There are a few networking requirements that must be satisfied for this to work. Of course, all server nodes must be able to talk to each other in both TCP and UDP. Otherwise, the gossip protocol as well as RPC forwarding will not work. 
-
-If Kong is to be used across datacenters, the network must be able to route traffic between IP addresses across regions as well. Usually, this means that all datacenters must be connected using a VPN or other tunneling mechanism. Kong does not handle VPN, address rewriting, or NAT traversal for you.
-
-Even if all the Kong nodes seem to be successfully part of a cluster, that doesn't mean they will be able to successfully communicate together: to check the status of the cluster and make sure the nodes can communicate with each other, you can run the [`kong cluster reachability`][cli-cluster] command.
-
-For multi-DC setups you will probably have to explicitly configure the [cluster_advertise][cluster_advertise] property on each node using an IP address and port that every Kong node (in any DC) can use to connect to that specific Kong node (in both TCP and UDP). For example if a node is available on DC1 at `1.1.1.1:7946` and another node is available at `2.2.2.2:7946` on DC2, the node on DC1 must have `cluster_advertise=1.1.1.1:7946` and the node on DC2 must have `cluster_advertise=2.2.2.2:7946`.
-
-## 4. Node Health States
+Kong nodes pointing to the same datastore will join in the same Kong cluster.
+Changes are propagated over the Kong cluster through events that are being
+registered in the datastore. Periodically all nodes will check for updates
+here and apply the changes.
```

Comment:
can we define "changes" in slightly more detail (e.g. updates to API definitions, plugin configurations, etc).